infrastructure/metrics/src/main/java/tech/pegasys/teku/infrastructure/metrics/SettableIntGauge.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.infrastructure.metrics;
+
+import java.util.concurrent.atomic.AtomicInteger;
+import org.hyperledger.besu.plugin.services.MetricsSystem;
+import org.hyperledger.besu.plugin.services.metrics.MetricCategory;
+
+public class SettableIntGauge {
```

Comment:
Is this distinction between `SettableIntGauge` and `SettableDoubleGauge` actually useful?  Looks like under the hood the ints are converted to doubles anyway ...

Reply:
You're right. It was unnecessary. Removed.